.github/workflows/packaging.yml

```diff
@@ -3,6 +3,15 @@ name: packaging
 on: [push, pull_request]
 
 jobs:
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Run action
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
+        with:
+          module-name: 'queue'
+          repository: 'tarantool/queue'
```

Comment:
How checkout works? We need to specify the repo inside the action, but doesn't need to specify PR commit?

Reply:
AFAIU, commit can be [specified](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsuses), but we can just use `@master` instead `@igrishnov/gh-204-add-hardcoded-versioning-support` here after done [PR](https://github.com/tarantool/actions/pull/54)

Reply:
No, I mean the checkout inside the action. Why we need to specify `tarantool/queue` repo, but do not required to specify commit to checkout?

Reply:
It looks like the `repository` variable can be [removed](https://github.com/actions/checkout/blob/main/README.md?plain=1#L24-L31), deleted